lib/internal/constants.dart

```diff
@@ -87,7 +87,7 @@ abstract class ArgumentName {
   static const String characteristicUuid = "characteristicUuid";
   static const String characteristicIdentifier = "characteristicIdentifier";
 
-  static const String bytes = "bytes";
+  static const String value = "value";
```

Comment:
Why this change and why in this PR?

Reply:
I created a separate PR for FlutterBleLib https://github.com/Polidea/FlutterBleLib/pull/232